guide-us-english.tex

```diff
@@ -298,11 +300,14 @@ \subsection{Continuous Improvement and Impediment Removal}
 
 \subsection{Cross-Team Coordination}
 
-The goals of Cross-Team Coordination are to:
+When multiple teams have to cooperate on the creation of a shared product,
+they have a strong need to coordinate.
+
+Cross-Team Coordination is essential in Scrum@Scale to:
```

Comment:
When a group of teams are in a SoS, they are by definition creating a common product, otherwise they wouldn't need to be in the same SoS. I'd rather see any description reflect that.